manifests/f/Funkin'Team/FridayNightFunkin'/v2.8/Funkin'Team.FridayNightFunkin'.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Funkin'Team.FridayNightFunkin'
+PackageVersion: v2.8
+PackageLocale: en-US
+Publisher: Funkin' Team
+PackageName: Friday Night Funkin'
+License: none
+ShortDescription: An open source rythym game written in Haxe
+ManifestType: defaultLocale
+ManifestVersion: 1.0.0
```

```suggestion
ManifestVersion: 1.4.0
```
Since changes are needed anyways